Resources/Prototypes/Datasets/tips.yml

```diff
@@ -1,137 +1,138 @@
-  - "When dealing with security, you can often get your sentence negated entirely through cooperation and deception."
-  - "Fire can spread to other players through touch! Be careful around flaming bodies or large crowds with people on fire in them."
-  - "Hull breaches take a few seconds to fully space an area. You can use this time to patch up the hole if you're confident enough, or just run away."
-  - "Burn damage, such as that from a welding tool or lightbulb, can be used to cauterize wounds and stop bleeding."
-  - "Bleeding is no joke! If you've been shot or acquired any other major injury, make sure to treat it quickly."
-  - "In an emergency, you can butcher a jumpsuit with a sharp object to get cloth, which can be crafted into gauze."
-  - "You can use sharp objects to butcher clothes or animals in the right click context menu. This includes glass shards."
-  - "Most explosives have an adjustable timer that you can set in the right click menu. This includes grenade penguins!"
-  - "You can stun grenade penguins, which can bide valuable time for you to kill them."
-  - "You can click on the names of items to pick them up in the right click menu, instead of hovering over the item and then selecting pick up."
-  - "Space Station 14 is open source! If there's a change you want to make, or a simple item you want to add, then try contributing to the game. It's not as hard as you'd think it is."
-  - "In a pinch, you can throw drinks or other reagent containers behind you to create a spill that can slip people chasing you."
-  - "Some weapons, such as knives & shivs, have a fast attack speed."
-  - "The jaws of life can be used to open powered doors."
-  - "If you're not a human, you can drink blood to heal back some of your blood volume, albeit very inefficiently."
-  - "If you're a human, don't drink blood! It makes you sick and you'll begin to take damage."
-  - "There is a chemical metabolism limit that limits the amount of reagents of a certain type you can digest at once. Certain species have higher metabolism limits, such as slimes."
-  - "Welding without proper eye protection can cause eye damage, which must be cured with oculine."
-  - "Zombies are very vulnerable to heat damage, making welding tools and laser guns extremely effective against them."
-  - "You can weld glass shards into glass sheets."
-  - "By right clicking on a player, and then clicking the heart icon, you can quickly examine them to check for injuries or how badly they're bleeding. You can also do this to yourself."
-  - "Monkeys and kobolds have a rare chance to be sentient. Ook!"
-  - "You can tell if an area with firelocks up is spaced by looking to see if the firelocks have lights beside them."
-  - "Instead of picking it up, you can alt-click food to eat it. This also works for mice and other creatures without hands."
+  - tips-dataset-1
+  - tips-dataset-2
```

Comment:
dont do this copy paste N times shit just have it be `type: dataset count: 135` for datasets that are required to be localized, at the very least

Reply:
yeah i would probably do something like this

Reply:
> dont do this copy paste N times shit just have it be `type: dataset count: 135` for datasets that are required to be localized, at the very least

Working on something very close to this.